Examples/Example/Example2_RandomizedSectionsAnimation.swift

```diff
@@ -13,19 +13,27 @@ import RxCocoa
 import CoreLocation
 
 class NumberCell : UICollectionViewCell {
-    @IBOutlet var value: UILabel?
+    @IBOutlet private var value: UILabel?
+
+    func configure(value: String) {
```

Comment:
I'd make these similar to this, to follow Apple naming conventions: 
```suggestion
    func configure(with value: String) {
```

Reply:
Sure, thanks, fixed